src/sw.ts

```diff
@@ -203,7 +204,10 @@ async function requestRouting (event: FetchEvent, url: URL): Promise<boolean> {
       status: 200
     }))
     return false
-  } else if (isConfigPageRequest(url)) {
+  } else if (isSubdomainConfigRequest(event)) {
+    log.trace('subdomain config request, ignoring and letting index.html handle it', event.request.url)
```

Comment:
note that this should never happen.. service worker isn't registered until after all the redirect dance is handled.

We may want to investigate migrating all the redirect logic added to index.tsx to the service worker.